doc/decisions/1_in_discussion/copy_on_write.md

```diff
@@ -12,9 +12,6 @@ There are also aspirations to create a new, simple [internal cache](../3_decided
 
 ## Constraints
 
-1. Elektra's public API should require no changes
-2. Modifying keys and keysets returned by `kdbGet` should work as expected
```

Comment:
I think it was removed, because I said in the last PR that "expected" is too vague. Without stating what is expected, this constraint is meaningless, everybody could add their own expectations.